src/words/EditButton.js

```diff
@@ -39,10 +48,33 @@ export default function EditButton({
       bookmark.context,
     );
     api.updateBookmark(bookmark.id, newWord, newTranslation, newContext);
+    if (newFitForStudy) {
+      api.setIsFitForStudy(bookmark.id);
```

Comment:
I think with Starring I am just thinking if it ever meant something different in the past. If it's functionality was always to override to force it to exercises then, a `userForceAddToExercises` could be a good name for it, as that is what it also did in the past. However, it might be weird to load in the "star" from the bookmark table in the future.

Alright, I can rename the front_end methods to something more generic. Maybe a simple _api.addToExercises_ and _api.removeFromExercises_ and keep the logic in the backend.